packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.ts

```diff
@@ -161,7 +161,7 @@ export function getGradableGraph(
     initialGraph: PerseusGraphType,
 ): PerseusGraphType {
     if (!state.hasBeenInteractedWith) {
-        return {type: initialGraph.type};
+        return {...initialGraph};
```

Comment:
I think this change is good. I manually tested it in the dev UI flipbook, and question grading still seems to work. Here is the grading logic that this is coupled to, from [interactive-graph.tsx](https://github.com/Khan/perseus/blob/322a2287bb6c4e3341e948d811769225e3d0949e/packages/perseus/src/widgets/interactive-graph.tsx)

```ts
const hasValue = Boolean(
    // @ts-expect-error - TS2339 - Property 'coords' does not exist on type 'PerseusGraphType'.
    userInput.coords ||
        // @ts-expect-error - TS2339 - Property 'center' does not exist on type 'PerseusGraphType'. | TS2339 - Property 'radius' does not exist on type 'PerseusGraphType'.
        (userInput.center && userInput.radius),
);

// The input wasn't correct, so check if it's a blank input or if it's
// actually just wrong
if (!hasValue || _.isEqual(userInput, rubric.graph)) {
    // We're where we started.
    return {
        type: "invalid",
        message: null,
    };
}
return {
    type: "points",
    earned: 0,
    total: 1,
    message: null,
};
```

Previously, this code was taking the `!hasValue` path to a score of `invalid`. Now, it's taking the `_.isEqual(userInput, rubric.graph)` path.